packages/core/framework/src/http/express-loader.ts

```diff
@@ -5,7 +5,11 @@ import session from "express-session"
 import Redis from "ioredis"
 import morgan from "morgan"
 import path from "path"
+import { logger } from "../logger"
 import { configManager } from "../config"
+import { MedusaRequest, MedusaResponse } from "./types"
+
+const NOISY_ENDPOINTS_CHUNKS = ["@fs", "@id", "@vite", "@react", "node_modules"]
```

Comment:
suggestion: maybe we can use a regexp directly? `new RegExp(/@fs|@id|@vite|@react|node_modules/, 'gm')`

Reply:
I think more of personal preference thing, or you think it will be faster?

Reply:
I think it will be faster on a big number of requests to not have to loop through the while array for every requests wdyt?

Reply:
some + includes is much better than regexp, lets not change that 👍